src/CMakeLists.txt

```diff
@@ -41,12 +41,15 @@ endif()
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_STANDARD 20)
 
+# Hotfix for boost bug (https://github.com/boostorg/asio/issues/312):
+add_compile_options(-DBOOST_ASIO_DISABLE_CONCEPTS)
```

Comment:
Is there a way to remind us to remove that option once in a while? Would a TODO help here?

Reply:
TODO sounds like something that can be done right now. Given the age of boost versions in Ubuntu, I'd guess that this won't make it into the standard boost any time soon,.

Reply:
Ack